resources/cdk/dynamodb-item-tracker/README.md

```diff
@@ -0,0 +1,103 @@
+# DynamoDB item tracker resources
+
+## Overview
+
+Creates AWS resources for Amazon DynamoDB item tracker sample applications: 
```

Comment:
Creates AWS resources --> Creates the following AWS resources

Reply:
fixed